.github/workflows/main.yml

```
@@ -0,0 +1,37 @@
+name: CI
```

maybe a bit more descriptive name for it?

```suggestion
name: CI rmw_iceoryx
```